Wikipedia/Code/UIViewController+Peekable.swift

```diff
@@ -0,0 +1,17 @@
+import UIKit
+
+extension UIViewController {
+    // Adds a peekable view controller on top of another view controller. Since the view controller is the one we want to show when the user peeks through, we're saving time by loading it behind the peekable view controller.
+    @objc func wmf_setupPeekable(_ peekableViewController: UIViewController, on viewController: UIViewController) -> UIViewController {
```

Comment:
you could remove the `onViewController` param and return value if you assume you're calling this on the VC that is getting the peekable added - `wmf_addPeekable(_` then use `self` instead of `viewController`